packages/mask/src/plugins/Avatar/Application/NFTListDialog.tsx

```diff
@@ -299,6 +299,12 @@ export function NFTListDialog(props: NFTListDialogProps) {
         return isGreaterThan(a.last_checked_at, z.last_checked_at) ? -1 : 1
     })
 
+    // Set eth to the default chain
+    useEffect(() => {
+        const defaultChain = first(chains)
+        if (!chains.includes(chainId) && defaultChain) setChainId(defaultChain)
```

Comment:
Don't set state in `useEffect`.

Reply:
The reason why we don't recommand this behaviour: https://mask.atlassian.net/wiki/spaces/F2E/pages/323158017/Set+a+state+in+useEffect